distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -87,6 +87,8 @@
 
     private static final String REMOTE_METACARD_ID = "09876543210987654321098765432100";
 
+    private static final long SLEEP_TIME = 2000;
```

Comment:
Just curious how you came up with the sleep time. In more resource constrained environments it shouldn't matter much since we have the timeout for 2 minutes, is that accurate?